Loading point clouds...
0it [00:00, ?it/s]
Traceback (most recent call last):
  File "/home/aniemcz/gooseReno/RENO/train.py", line 82, in <module>
    dataflow = torch.utils.data.DataLoader(
  File "/home/aniemcz/gooseReno/.pixi/envs/default/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 350, in __init__
    sampler = RandomSampler(dataset, generator=generator)  # type: ignore[arg-type]
  File "/home/aniemcz/gooseReno/.pixi/envs/default/lib/python3.10/site-packages/torch/utils/data/sampler.py", line 143, in __init__
    raise ValueError(f"num_samples should be a positive integer value, but got num_samples={self.num_samples}")
ValueError: num_samples should be a positive integer value, but got num_samples=0
Traceback (most recent call last):
  File "/home/aniemcz/gooseReno/RENO/train.py", line 82, in <module>
    dataflow = torch.utils.data.DataLoader(
  File "/home/aniemcz/gooseReno/.pixi/envs/default/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 350, in __init__
    sampler = RandomSampler(dataset, generator=generator)  # type: ignore[arg-type]
  File "/home/aniemcz/gooseReno/.pixi/envs/default/lib/python3.10/site-packages/torch/utils/data/sampler.py", line 143, in __init__
    raise ValueError(f"num_samples should be a positive integer value, but got num_samples={self.num_samples}")
ValueError: num_samples should be a positive integer value, but got num_samples=0
